src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -297,13 +299,41 @@ public function viewAction(int $orderId, Request $request): Response
             'addOrderPaymentForm' => $addOrderPaymentForm->createView(),
             'changeOrderCurrencyForm' => $changeOrderCurrencyForm->createView(),
             'privateNoteForm' => $privateNoteForm->createView(),
-            'addProductToOrderForm' => $addProductToOrderForm->createView(),
-            'updateOrderProductForm' => $updateOrderProductForm->createView(),
             'updateOrderShippingForm' => $updateOrderShippingForm->createView(),
             'invoiceManagementIsEnabled' => $orderForViewing->isInvoiceManagementIsEnabled(),
         ]);
     }
 
+    /**
+     * @param int $orderId
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function addProductAction(int $orderId, Request $request): Response
```

Comment:
You need to add the `@AdminSecurity` on ALL the actions missing it For json actions ideally you would need a redirect_route returning a json response with 403 code

Reply:
:ballot_box_with_check: